src/components/UpdateLanguage.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from "react";
+import { Menu } from "semantic-ui-react";
+import "../styling/Navbar.css";
+import i18n from "../i18n";
+
+class UpdateLanguage extends Component {
+  render() {
+    return (
+      <>
+      <div id="language" className="ui simple dropdown item">
```

Comment:
If you use a `div` to wrap the entire JSX you return, then you don't need a `React.fragment`

Reply:
That is correct.  Fixed.